openmls/Cargo.toml

```diff
@@ -27,6 +27,7 @@ itertools = {version = "0.10", optional = true}
 default = ["rust-crypto"]
 rust-crypto = ["evercrypt/rust-crypto-aes"]
 expose-test-vectors = ["itertools"]
+test_utils = []
```

Comment:
There's no clear guideline on this yet ([see for example this discussion](https://github.com/rust-lang/api-guidelines/discussions/101)). But all other features are kebab case. So we should probably stick to it

Reply:
Right you are. That was really just a typo.